boards/shields/rk055hdmipi4m/Kconfig.defconfig

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_RK055HDMIPI4M
+
+config DISPLAY
+	default y
```

Comment:
This should be set at application level, not as part of default configuration.
Same policy used in boards apply here.

Reply:
@danieldegrasse This does not seem to be addressed.

Reply:
^^